Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -823,11 +830,15 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
                 try? await updateTunnelConfiguration(serverSelectionMethod: serverSelectionMethod)
                 completionHandler?(nil)
             }
-        case .setIncludeAllNetworks,
+        case .setConnectOnLogin,
+                .setIncludeAllNetworks,
                 .setEnforceRoutes,
                 .setExcludeLocalNetworks,
+                .setNotifyStatusChanges,
                 .setRegistrationKeyValidity,
-                .setSelectedEnvironment:
+                .setSelectedEnvironment,
+                .setShowInMenuBar,
+                .setShowVPNSettings:
```

Comment:
Added several new options that don't need handling in the extension.

Still it's cool that the settings mechanism allows us to handle these if anything changed.  Like for example if eventually the VPN settings could be directly changed from the extension.  Or if we wanted to fire a pixel.

Reply:
Nice :+1: